config/config.go

```diff
@@ -54,22 +53,6 @@ type AttrConfig struct {
 	DeleteProcessedData *bool             `json:"delete_processed_data"`
 }
 
-// NewAttrConfig creates a SLAM config from a service config.
-func NewAttrConfig(cfg config.Service) (*AttrConfig, error) {
```

Comment:
[note to self] This is dead code. We were planning to use it [here](https://github.com/viamrobotics/viam-orb-slam3/pull/29/files#diff-c70dd5ed993c1b53cd893d5a38fa4665c771456384a369b2a67b7a55ddb9fffeR93) but there is now a canonical function to use to convert the json into a slam AttrConfig.